aws/resource_aws_fsx_lustre_file_system_test.go

```diff
@@ -786,5 +826,43 @@ resource "aws_fsx_lustre_file_system" "test" {
   deployment_type             = "PERSISTENT_1"
   per_unit_storage_throughput = %[1]d
 }
+
+data "aws_kms_alias" "fsx" {
+  name = "alias/aws/fsx"
+}
```

Comment:
Is this KMS Key guaranteed to exist in AWS accounts that have not provisioned FSx before? Generally, AWS-managed keys are not created until then. It might be best to remove this here and update the test step to just check for a regular expression so there is not a test dependency the first run in a new account.

```suggestion
```

Reply:
that is a valid reason, either we can add regex match or just remove the check, since it has to be kms by default and we don't need to verify it.(how it was before)

Reply:
I was able to confirm in a separate AWS account that the FSx KMS Key and its associated Alias is not automatically available. 👍

Reply:
Updated